gobblin-iceberg/src/main/java/org/apache/gobblin/iceberg/writer/IcebergMetadataWriter.java

```diff
@@ -637,6 +644,7 @@ protected void addFiles(GobblinMetadataChangeEvent gmce, Map<String, Collection<
   private Stream<DataFile> getIcebergDataFilesToBeAddedHelper(GobblinMetadataChangeEvent gmce, Table table,
       Map<String, Collection<HiveSpec>> newSpecsMap,
       TableMetadata tableMetadata) {
+    tableMetadata.serializedAuditCountMaps.add(gmce.getAuditCountMap());
```

Comment:
See my below comment about collecting audit counts only if the table is whitelisted